src/jsMain/kotlin/SignupComponent.kt

```diff
@@ -1,20 +1,204 @@
+                    onSubmit = { input ->
+                        setUsername(input)
+                    }
+                    onChange = { input ->
+                        setUsername(input)
+                    }
+                    type = "text"
+                }
+        )}
+    }
+    styledDiv{
+        css{
+            margin(vertical = 20.px)
+            textAlign=TextAlign.center
+        }
+        label{
+            +"Enter Password: "
+
+        child(
+                InputComponent,
+                props = jsObject {
+                    onSubmit = { input ->
+                        setPassword(input)
+                    }
+                    onChange = { input ->
+                        setPassword(input)
+                    }
+                    type = "text"
+                }
+        )}
+    }
+    styledDiv{
+        css{
+            margin(vertical = 20.px)
+            textAlign=TextAlign.center
+        }
+        label {
+            +"Size: "
+            select {
+                attrs.onChangeFunction = {
+                    val target = it.target as HTMLSelectElement
+                    setSize(target.value)
+                }
+                option {
+                    attrs.value = "Adult"
+                    +"Adult"
+                }
+                option {
+                    attrs.value = "Kids"
+                    +"Kids"
+                }
+            }
+        }
+    }
+    styledDiv{
+        css{
+            margin(vertical = 20.px)
+            textAlign=TextAlign.center
+        }
+        label{
+            +"Gender: "
+
+        select{
+            attrs.onChangeFunction = {
+                val target = it.target as HTMLSelectElement
+                setGender(target.value)
+            }
+            option{
+                attrs.value = "Female"
+                +"Female"
+            }
+            option{
+                attrs.value = "Male"
+                +"Male"
+            }
+            option{
+                attrs.value = "Unisex"
+                +"Unisex"
+            }
+        }}
+
+    }
+    styledDiv{
+        css{
+            margin(vertical = 20.px)
+            textAlign=TextAlign.center
+        }
+        label{
+            +"Enter Zipcode: "
+        child(
+                InputComponent,
+                props = jsObject {
+                    onSubmit = { input ->
+                        setZipcode(input)
+                    }
+                    onChange = { input ->
+                        setZipcode(input)
+                    }
+                    type = "text"
+                }
+        )}
+    }
+    styledDiv{
+        css{
+            margin(vertical = 20.px)
+            textAlign=TextAlign.center
+        }
+        button{
+        +"Sign up"
+        attrs.onClickFunction = {
+            render(document.getElementById("root")) {
+                child(LoginPage)
+            }
+        }
+    }}
+    /*p{
```

Comment:
Should this be here or be commented out?

Reply:
It should be there.